src/core/server/models/user/helpers.ts

```diff
@@ -4,14 +4,10 @@ import { SSOUserProfile } from "coral-server/app/middleware/passport/strategies/
 
 import { GQLUSER_ROLE } from "coral-server/graph/schema/__generated__/types";
 
+import { isSiteModerationScoped } from "coral-common/permissions";
```

Comment:
Can you please include this import up with that from `coral-server` in the same grouping?